### ISSUE TITLE
Elevated circulating glucosylsphingosine levels 

### ISSUE THREAD

**Issue by carolina-f** [obophenotype/human-phenotype-ontology#10911]
**Preferred term label:**

**Synonyms**
Elevated circulating GlSph levels  

**Definition (free text, please give PubMed ID)**
PMID: 28457694 Gaucher cells identified in his bone marrow and the highly elevated plasma chitotriosidase activity and **glucosylsphingosine** supported a diagnosis of Gaucher disease. However, the leukocyte β-glucosidase activity was in a normal range.
(...) Based on our findings, saposin C deficiency should be considered in view of the normal glucocerebrosidase activity and high level of chitotriosidase and **glucosylsphingosine** in patients with unexplained hepatosplenomegaly.

PMID: 26831127 LysoGb3 is particularly high in Fabry disease, while a markedly increased level of SPC is observed in Niemann-Pick type A/B, and a **high GlSph** content is specific of Gaucher disease. Remarkably, both LysoGb3 and **GlSph** were found to be increased in the two patients with PSAP deficiency.
(...) Molecular analysis and plasma lysoSLs profile, resulting in an increased amount of LysoGb3 and **GlSph**, were essential for diagnosis of PSAP deficiency.

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Atypical Gaucher disease due to saposin C deficiency ORPHA:309252

**Your nano-attribution (ORCID)**
0009-0005-6714-5727

**Comment by pnrobinson** [comment]
@carolina-f 
Adding new term: 
- ID: HP:6001185
- label: Elevated circulating glucosylsphingosine concentration
- parents: [Abnormal circulating carbohydrate concentration (HP:0011013)](https://hpo.jax.org/app/browse/term/HP:0011013)
- pmid: PMID:39881421
- def: Concentration of D-glucosylsphingosine in the blood circulation above the upper limit of normal.
- comment: Glucosylsphingosine (GlcSph) is a key biomarker of Gaucher Disease.
- orcid: orcid:0009-0005-6714-5727
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10911
- synonyms:
